lang/rapidjson/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem                      1.0
+PortGroup                       github 1.0
+PortGroup                       cmake 1.1
+
+github.setup                    Tencent rapidjson b32cd9421c5e3cbe183a99b6537ce11441e50656
+version                         20180424
```

Comment:
Why a snapshot from master and not a [release](https://github.com/Tencent/rapidjson/releases)?

Reply:
Last release is like 2 years old.

Reply:
Upstream must have its reasons not to provide a newer release. Ports are supposed to provide stable versions. A prerelease following master could additionally be provided by a port with the`-devel` suffix.

Reply:
@raimue whatever the reason, 1.1.0 fails to build most examples and tests, and will not work with c++11.

Reply:
If there is no way to make 1.1.0 work, this port should be named rapidjson-devel to indicate it is not following releases.

Reply:
Will do. I spent all day yesterday trying to get 1.1.0 to work and eventually gave up. Obviously, if there's a new release in the future that works, I'd fix the port to use that.